unlock-app/src/components/creator/lock/KeyList.js

```diff
@@ -5,15 +5,14 @@ import UnlockPropTypes from '../../../propTypes'
 import { expirationAsDate } from '../../../utils/durations'
 import Pagination from '../../interface/pagination/Pagination'
 
-// TODO add pagination
 export function KeyList({ keys }) {
-  const renderItems = items => {
-    return items.map(item => {
+  const renderItems = lockKeys => {
+    return lockKeys.map(lockKey => {
```

Comment:
nit: destructure? `return lockKeys.map(({id, transaction, expiration, data})=> {` and then use them directly below?